app/controllers/notifications_controller.rb

```diff
@@ -75,6 +75,11 @@ def index
     check_out_of_bounds(scope)
 
     @total = scope.count
+
+    @inbox_notification    = current_user.notifications.inbox.count
+    @starred_notification  = current_user.notifications.starred.count
+    @archived_notification = current_user.notifications.archived.count
+
```

Comment:
I wonder if we can reduce the amount of sql queries here with a group on `:archived`